redwood-lazylayout-uiview/src/commonMain/kotlin/app/cash/redwood/lazylayout/uiview/UIViewLazyList.kt

```diff
@@ -144,6 +152,9 @@ internal open class UIViewLazyList() : LazyList<UIView>, ChangeListener {
         collectionView: UICollectionView,
         numberOfItemsInSection: NSInteger,
       ): NSInteger {
+        // NB: This is a bit of a hack, but we don't want to let the items list know
+        // the collectionView's frame until it's had a layout pass.
+        items.containerFrame = collectionView.frame()
```

Comment:
This method gets called a lot, can we cache this value and only set it once, or better yet when the frame changes?

Reply:
Good point, I'll move this to the `width`/`height` hooks instead

Reply:
Better yet, I'll just cache a reference to the `collectionView` itself on load, and check its frame every time we insert a widget